pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -190,6 +190,11 @@ func (m *kubeGenericRuntimeManager) generateContainerConfig(container *v1.Contai
 	}
 
 	command, args := kubecontainer.ExpandContainerCommandAndArgs(container, opts.Envs)
+	logDir := fmt.Sprintf("%s/%s", buildPodLogsDirectory(pod.UID), container.Name)
```

Comment:
Use your helper function. `BuildContainerLogsDirectory`

Reply:
Remove this directory when remove container.

Reply:
good point. Will do